Cargo.toml

```diff
@@ -15,7 +15,7 @@ build = "src/gl/build.rs"
 name = "gl"
 path = "src/gl/lib.rs"
 
-[build-dependencies.gl_generator]
+[dependencies.gl_generator]
```

Comment:
Oh, that's because I forgot to remove `extern crate gl_generator;` from the tests.